docs/testplan/loss_threshold_test.md

```diff
@@ -0,0 +1,22 @@
+# SONiC Switch Loss Threshold Test
+
+- [SONiC Switch Loss Threshold Test](#sonic-switch-loss-threshold-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Test Steps](#test-steps)
+
+## Test Objective
+
+This test aimes to determine the maximum traffic rate that results in 0% packet loss across different packet sizes. By identifying this threshold, we can assess the switch's forwarding capability and validate its performance under various traffic conditions.
+
+## Test Setup
+
+The test is designed to be topology-agnostic, meaning it does not assume or impose a specific network connection. The only requirement is that the DUT is fully connected to handle full traffic loads under stress. Plus all traffic generators in the testbed must be time synchronized.
+
+## Test Steps
+
+1. Establish the required IPv6 BGP sessions and IPv6 routes by following the procedures outlined in the Snappi BGP test plan and the IPv6 route scaling test plan.
```

Comment:
If other test plan is referred, better add a link here.

Also, this is more like a testbed setup phase, so will be better to move to the previous chapter.